_partials/_graphing-ohlcv-data.md

```diff
@@ -0,0 +1,33 @@
+## Graph OHLCV data
+
+When you have extracted the raw OHLCV data, you can use it to graph the result
+in a candlestick chart, using Grafana. To do this, you need to have Grafana set
+up to connect to your TimescaleDB database. For more information about how to do
+this, see the [Grafana setup instructions][grafana-setup].
+
+<Procedure>
+
+### Graphing OHLCV data
+
+1.  Ensure you have Grafana installed, and you are using the TimescaleDB
+    database that contains the Twelve Data stocks dataset set up as a
+    data source. For more information about how to do this, see the [Grafana
+    setup instructions][grafana-setup].
+1.  In Grafana, from the `Dashboards` menu, click `New Dashboard`. In the
+    `New Dashboard` page, click `Add a new panel`.
```

Comment:
```suggestion
1.  Add a new panel for the tutorial. From the `Dashboards` menu in Grafana, click `New Dashboard` and then click `Add a new panel`.
```

Reply:
Style guide: The place of the action, then the action.